plugins/scaffolder-backend/src/scaffolder/actions/builtin/fetch/cookiecutter.test.ts

```diff
@@ -100,11 +102,17 @@ describe('fetch:cookiecutter', () => {
   it('should execute the cookiecutter templater with the correct values', async () => {
     await action.handler(mockContext);
 
+    const values = {
+      ...mockContext.input.values,
+      _copyWithoutRender: ['./github/workflows/*'],
```

Comment:
this should be `_copy_without_render` https://cookiecutter.readthedocs.io/en/1.7.2/advanced/copy_without_render.html

Reply:
Oops 😳 Will fix shortly.